k8s.gcr.io/images/k8s-staging-networking/images.yaml

```diff
@@ -13,3 +13,7 @@
 - name: ip-masq-agent-ppc64le
   dmap:
     "sha256:fd7449efcd712a85dbcdf6c1648f5f9bb674c208862f469c457ccd154d7d12bf": ["v2.6.0"]
+-name: ingress-gce-404-server-with-metrics-amd64:
```

Comment:
It would be nice if we built these as manifest-lists so the architecture was not part of the name.

Is this really GCE specific or just happens to be used by GCEs ingress?

Reply:
This is specific to GCEs ingress. The naming predates me. 
Any suggestions on the name?

Reply:
I see the name existing already, so I'll OK this. :+1: 

If you want to switch to manifest-lists, I can show you some examples.